templates/microsoft-SQLServer2017/README.md

```diff
@@ -0,0 +1,21 @@
+Microsoft SQL Server 2017 Database Monitoring
+=============================================
+
+Monitors the Microsoft SQL Server Database installed on a Windows server.
+
+Provides monitoring of:
+* Logs
+* Data Directory
+* Events
+* Performance metrics.
+* Running Processes
+* Services
+
+
+In addition, you can add SQL queries and process the results using the "Analysis" feature.
+
+* Import the cfg and activate it on the Host where the SQL Server Database is installed.
+* Set the System Credentials to access the server
```

Comment:
System Credentials and sq_sa Credentials are all part of the Host Settings, which is described on line 21. Please rework this part. Also, is it sq_sa or sql_sa ?